mods/mc_worldmanager/commands.lua

```diff
@@ -29,13 +23,23 @@ minetest.register_chatcommand("localPos", {
 
 commands["new"] = {
     func = function(name, params)
-        local realmName = params[2]
+        local realmName = tostring(params[1])
         if (realmName == "" or realmName == nil) then
```

Comment:
[Same issue with unintended `"nil" == nil` comparison](https://github.com/ubc-minetest-classroom/Minetest_Classroom/pull/100#discussion_r910437827) as in #100, due to `tostring(nil)` producing `"nil"`
```suggestion
        if (realmName == "" or realmName == "nil") then
```
```suggestion
        if (realmName == "" or realmName == params[1]) then
```